rmgpy/kinetics/arrhenius.pyx

```diff
@@ -227,10 +227,13 @@ cdef class Arrhenius(KineticsModel):
         """
         self._A.value_si *= factor
 
-    def to_cantera_kinetics(self):
+    def to_cantera_kinetics(self, ArrObj=False):
```

Comment:
not a fan of the name `ArrObj`. Too abbreviated, not used elsewhere, and not in "house style". Maybe `cantera_Arrhenius` or `arrhenius_class`? 

Also the docstrings need further work. Currently says "converts to  cantera Arrhenius object", and "Arrhenius(A,b,E)"  instead of referring to "ArrheniusRate"

Reply:
changing to `arrhenius_class`.

Reply:
how about this for a docstring on "to_cantera_kinetics"

```
"""
        Converts the RMG Arrhenius object to a cantera ArrheniusRate or
        the auxiliary cantera Arrhenius (used by falloff reactions). 
        Inputs for both are (A,b,E)  where A is in units of m^3/kmol/s, b is dimensionless, and E is in J/kmol

        arrhenius_class: If true, uses cantera.Arrhenius (for falloff reactions). If false, uses 
        Cantera.ArrheniusRate
"""
```